extensions/opentelemetry/deployment/src/main/java/io/quarkus/opentelemetry/deployment/logging/LogHandlerProcessor.java

```diff
@@ -18,6 +33,57 @@
 @BuildSteps(onlyIf = LogHandlerProcessor.LogsEnabled.class)
 class LogHandlerProcessor {
 
+    private static final DotName LOG_RECORD_EXPORTER = DotName.createSimple(LogRecordExporter.class.getName());
+    private static final DotName LOG_RECORD_PROCESSOR = DotName.createSimple(LogRecordProcessor.class.getName());
+    //    private static final DotName LOG_RECORD_HANDLER = DotName.createSimple(Handler.class.getName());
+
+    @BuildStep
+    UnremovableBeanBuildItem ensureProducersAreRetained(
```

Comment:
`UnremovableBeanBuildItem` ensures that a bean is retained by Arc. If the issue is with a native build, this is not what we need here.
Moreover, OpenTelemetryLogHandler is not a CDI bean so it will not have any effect.